milvus/grpc/BaseClient.ts

```diff
@@ -20,35 +20,37 @@ const schemaProtoPath = path.resolve(
  * Base gRPC client, setup all configuration here
  */
 export class BaseClient {
-  // The gRPC client instance.
-  client: Client | undefined;
+  // metadata
```

Comment:
Could we have more details about metadata? It is unclear where we have used it or how to use it.